app/uploaders/banner_image_uploader.rb

```diff
@@ -0,0 +1,8 @@
+class BannerImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::Compatibility::Paperclip
```

Comment:
Do we need `CarrierWave::Compatibility::Paperclip`? I'm not seeing any other use of `paperclip` off-hand.